builder/src/non_permissioned.rs

```diff
@@ -195,7 +195,7 @@ impl BuilderConfig {
         let global_state_clone = global_state.clone();
         tracing::info!("Running permissionless builder against hotshot events API at {events_url}",);
         async_spawn(async move {
-            let res = run_non_permissioned_standalone_builder_service::<_, V>(
+            let res = run_non_permissioned_standalone_builder_service::<_, StaticVersion<0, 1>>(
```

Comment:
please merge main into this branch. We are now passing `SequencerApiVersion` here